installing/installing_restricted_networks/installing-restricted-networks-aws.adoc

```diff
@@ -67,6 +67,12 @@ include::modules/ssh-agent-using.adoc[leveloffset=+1]
 
 include::modules/installation-generate-aws-user-infra.adoc[leveloffset=+1]
 
+include::modules/installation-generate-aws-user-infra-install-config.adoc[leveloffset=+2]
+
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+
+include::modules/installation-generate-aws-user-infra-ignition.adoc[leveloffset=+2]
+
 // After the proxy change merges, I need to put it in and emphasize that you
```

Comment:
You can remove this note, too.

Reply:
@kalexand-rh I can remove it, but just wanted to make sure we didn't need to add anything for the "emphasize that you must configure a proxy..." as the note says?